### PR TITLE
Homework_4_Roza_Yeghiazaryan

### PR DESCRIPTION
**Pull request by RosaRoing**

### CODE REVIEW

File: src/algorithms/SortingAlgorithms.java [hunk 1]
```diff
@@ -19,11 +21,30 @@ public static void sortByBubbleSorting(int[] array) {
     }
 
     public static void sortBySelectionSorting(int[] array){
```
Comment (vahan1994vs) [inline src/algorithms/SortingAlgorithms.java:23]: It doesn't work correctly
https://en.wikipedia.org/wiki/Selection_sort

**Review by vahan1994vs — COMMENTED** [review]
Reviwed

File: src/algorithms/SortingAlgorithms.java (hunk 1 above)
Comment (RosaRoing) [reply src/algorithms/SortingAlgorithms.java:23]: Thanks for valuable comment!

**Comment by vahan1994vs** [comment]
needs work

**Pull request closed without merging**